api/src/main/java/org/openmrs/api/OrderService.java

```diff
@@ -85,7 +85,7 @@
 	 * @should pass if an active order for the same concept exists in a different care setting
 	 * @should set Order type of Drug Order to drug order if not set and concept not mapped
 	 * @should set Order type of Test Order to test order if not set and concept not mapped
-	 * @should fail if an active drug order for the same drug formulation exists
+	 * @should throw AmbiguousOrderException if an active drug order for the same drug formulation exists
 	 * @should pass if an active order for the same concept exists in a different care setting
```

Comment:
You also need a test to ensure that it passes for overlapping orders in case the parallel orders are specified